modules/core/src/main/java/org/apache/ignite/internal/visor/tx/VisorTxTask.java

```diff
@@ -310,9 +324,12 @@ else if (locTx instanceof GridDhtTxRemote) {
                     killClo = REMOTE_KILL_CLOSURE;
                 }
 
-                infos.add(new VisorTxInfo(locTx.xid(), locTx.startTime(), duration, locTx.isolation(), locTx.concurrency(),
-                    locTx.timeout(), lb, mappings, locTx.state(),
-                    size, locTx.nearXidVersion().asGridUuid(), locTx.masterNodeIds(), locTx.topologyVersionSnapshot()));
+                TxVerboseInfo verboseInfo = arg.txInfoArgument() == null ? null : createVerboseInfo(ignite, locTx);
+
+                infos.add(new VisorTxInfo(locTx.xid(), locTx.startTime(), duration, locTx.isolation(),
+                    locTx.concurrency(), locTx.timeout(), lb, mappings, locTx.state(), size,
+                    locTx.nearXidVersion().asGridUuid(), locTx.masterNodeIds(), locTx.topologyVersionSnapshot(),
+                    verboseInfo));
```

Comment:
isVerboseMode? create: null
What do you think?

Reply:
See above.